doc/examples/brain_extraction_dwi.py

```diff
@@ -3,8 +3,8 @@
 Brain segmentation with median_otsu
 ===================================
 
-We show how to extract brain information and mask from a b0 image using dipy's
-segment.mask module.
+We show how to extract brain information and mask from a b0 image using dipy_'s
```

Comment:
I'd dare to say that the incorrect links to DIPY (`dipy_`) were due to the missing include `links_names.inc`. I've included it in the reported files. See whether this solves the issue.